components/home/Home.jsx

```diff
@@ -1,10 +1,101 @@
 import * as React from 'react';
-import { Text, View } from 'react-native';
-
-export default (props) => {
-   return (    
-      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-         <Text>Home Screen</Text>
-         <Text>Replace these Text tags with your component</Text>
-      </View>);
-}
\ No newline at end of file
+import { View, Text, StyleSheet } from 'react-native';
+import { Button, Title, Subheading} from 'react-native-paper';
+
+{/* Temporary workout variables*/}
+let workout1 = "Workout 1"
+let workout2 = "Workout 2"
+
+export default class App extends React.Component {
+
+   render() {
+      return (
+         <View style = {styles.container}>
+
+
+            {/*Cycles Section*/}
+            <View style = {{...styles.sectionComponent, margin: 30}}>
+
+               <Title style = {styles.title}>
+                  Cycles
+               </Title>
+
+               <Button contentStyle={{width: 331, height: 170}} style = {[{backgroundColor: "#5DB075", borderRadius: 20, borderColor: "#5DB075", position: "absolute", top: 60, alignSelf: "center"}]} color = "#FFFFFF" onPress = {() => alert("This should start the current workout in the cycle")}>
+                  Start Cycle Workout
+               </Button>
+
+               <Button contentStyle = {{width: 325, height: 35}} style = {styles.seeAllButton} color = "#000000" onPress = {() => alert("This should take you to workouts tab")}>
+                  See all cycles
+               </Button>
+               
+            </View>
+
+
+            {/*Workouts Section*/}
+            <View style = {{...styles.sectionComponent, margin: 10}}>
+               <Title style = {styles.title}>
+                  Workouts
+               </Title>
+
+               <Subheading style = {[{position: "absolute", top: 82, left: 10, fontSize: 18}]}>
+                  Workout 1
+               </Subheading>
+
+               <Subheading style = {[{position: "absolute", top: 157, left: 10, fontSize: 18}]}>
+                  Workout 2
+               </Subheading>
```

Comment:
So you said that these workout will be dynamic based off a list. Try doing the same thing you did with the variables at the top with this section but use a list instead in case we have 0 or 1 workouts.